DashAI/back/database/models.py

```diff
@@ -81,9 +81,15 @@ class Run(Base):
     status: Mapped[Enum] = mapped_column(
         Enum(RunStatus), nullable=False, default=RunStatus.NOT_STARTED
     )
+    delivery_time: Mapped[DateTime] = mapped_column(DateTime, nullable=True)
     start_time: Mapped[DateTime] = mapped_column(DateTime, nullable=True)
     end_time: Mapped[DateTime] = mapped_column(DateTime, nullable=True)
 
+    def run_delivery(self):
```

Comment:
Why not to call the state changer method with a slightly more descriptive name as `set_status_as_delivered`?
This change could be easily extended to the remaining state changing methods. (`set_status_as_started` and `set_status_as_finished`).

```suggestion
    def set_status_as_delivered(self):
```